apps/docusaurus/docs/move/move-on-aptos/move-security-guidelines.md

```diff
@@ -12,7 +12,65 @@ This guide addresses this gap by detailing common anti-patterns and their secure
 
 ---
 
-### Object Access Control
+### Object Ownership Check
+
+Every `Object<T>` can be accessed by anyone, which means any `Object<T>` can be passed to any function, even if the caller doesn't own it.
+It's important to verify that the `signer` is the rightful owner of the object.
+
+#### Example Insecure Code
+
+In this example, a user must make a payment before being able to perform actions within the module. The user needs to invoke `buy_action` to purchase an `Object<Action>`, which they can then use for executing operations later on.
+
+```move
+module user::obj{
+
+    public entry fun buy_action(user:&signer,amount_of_action: u64){
+        let price = calc_price(amount_of_action);
+        deposit(user,price);
+
+        let user_address = address_of(user);
+        let constructor_ref = object::create_object(user_address);
+        let object_signer = object::generate_signer(&constructor_ref);
+        move_to(&object_signer, Action {amount_of_action });
+    }
+
+    public entry fun perform_action(user:&signer,obj: Object<Action>){
```

Comment:
it's not clear what this code does or why it matters that the ownership of the object is checked.